authors/apps/authentication/views.py

```diff
@@ -1,15 +1,18 @@
 from rest_framework import status
-from rest_framework.generics import RetrieveUpdateDestroyAPIView
-from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.generics import RetrieveUpdateDestroyAPIView, RetrieveUpdateAPIView
```

Comment:
RetrieveUpdateDestroyAPIView, I believe you're not using this.

Reply:
You have a linting eye @codeplus254

Reply:
I am using it on the code I am working on. I pushed that code so that you could have something to work on

Reply:
I rebased @ThaDeveloper work. He was the one working on views